apps/ledger-live-desktop/tests/models/AddAccountModal.ts

```diff
@@ -23,11 +26,14 @@ export class AddAccountModal extends Modal {
     this.stopButton = page.locator("data-test-id=add-accounts-import-stop-button");
     this.addMoreButton = page.locator("data-test-id=add-accounts-finish-add-more-button");
     this.doneButton = page.locator("data-test-id=add-accounts-finish-close-button");
+    this.selectedCurrency = currency => page.locator(`text=${currency}`);
+    this.addNewAccount = page.locator("text=Add new account");
```

Comment:
is it button? if yes could you put it in the name of the variable?

Reply:
Variable was actually useless. I just removed it